images/win/windows2019.json

```diff
@@ -125,7 +124,10 @@
             ],
             "scripts": [
                 "{{ template_dir }}/scripts/Installers/Install-PowerShellModules.ps1",
+                "{{ template_dir }}/scripts/Installers/Install-WindowsFeatures.ps1",
+                "{{ template_dir }}/scripts/Installers/Install-Choco.ps1",
                 "{{ template_dir }}/scripts/Installers/Initialize-VM.ps1",
+                "{{ template_dir }}/scripts/Installers/Configure-Antivirus.ps1",
```

Comment:
Maybe move this to the very beginning to avoid possible slowdowns related to the antivirus? Or is it not possible as we need something from Initialize-vm step?